basics/patching.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: contribute
+title: Patch acceptance process
+---
+
+# Patch Acceptance Process
+
+1. Read the [Bazel governance plan](/governance.html).
+1. Discuss your plan and design, and get agreement on a GitHub issue or our
+   [mailing list](https://groups.google.com/forum/#!forum/bazel-dev).
+1. PRs that change or add behavior are not accepted without being tied to an
```

Comment:
This isn't a "step" but rather a rationale for the previous step. So fold it into that one.

Reply:
Done.